modules/coreos-layering-configuring.adoc

```diff
@@ -30,12 +30,14 @@ For example, the following Containerfile creates a custom layered image from an
 [source,yaml,subs="+attributes"]
 ----
 # Using a {product-version}.0 image
-FROM quay.io/openshift-release/ocp-release@sha256... <1>
-#Install hotfix rpm
-RUN rpm-ostree cliwrap install-to-root / && \ <2>
-    rpm-ostree override replace http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/kernel-{,core-,modules-,modules-core-,modules-extra-}5.14.0-295.el9.x86_64.rpm && \ <3>
-    rpm-ostree cleanup -m && \
-    ostree container commit
+containerfileArch: noarch
```

Comment:
issue: I don't think that this would work like this. Or at the very least, if we instruct people to do it this way, we'll need to keep in mind that their final image would not have the MachineConfigs baked into it. That's why one uses the `FROM configs AS final` line; so that they can get the image that we've already layered things onto.

For the out-of-cluster scenario, they still need to set the `osImageURL` field on a MachineConfig for the time-being.